client/pages/BoardPage.jsx

```diff
@@ -48,6 +48,21 @@ function BoardPage({user}) {
       getBoard();
     }, [])
 
+    const navigate = useNavigate();
+
+    async function logout() {
+      try {
+        // make DB call to terminate session
+        await axios.delete('/logout');
+        // clear userState
+        setUser(null);
+        // navigate to login page
+        navigate('/login');
+      } catch(err) {
+        console.log('error in BoardPage.jsx logout function: ', err.message)
+      }
+    }
```

Comment:
Should we move this to nav bar now?

Reply:
For sure.  This brings up another workflow question that I just posted in slack

Reply:
i'm aware of this change.  I'm working on a current branch that will implement this change.  lets merge this PR as is and I'll address that in the next PR